apps/web/src/actions/copilot/refinePrompt.ts

```diff
@@ -76,6 +76,7 @@ export const refinePromptAction = authProcedure
     }).then((r) => r.unwrap())
 
     const result = await sdk.run(env.COPILOT_REFINE_PROMPT_PATH, {
+      stream: true,
       parameters: {
         prompt: document.content,
         evaluation: evaluation.metadata.prompt,
```

Comment:
This should not be required, right? It's awaiting for the result anyways

Reply:
What should be required? the `stream` argument?